json-format.md

```diff
@@ -142,11 +141,9 @@ Example event with `String`-valued `data`:
     "source" : "/mycontext",
     "eventID" : "A234-1234-1234",
     "eventTime" : "2018-04-05T17:31:00Z",
-    "extensions" : {
-      "comExampleExtension" : "value"
-    },
     "contentType" : "text/xml",
-    "data" : "<much wow=\"xml\"/>"
+    "data" : "<much wow=\"xml\"/>",
```

Comment:
It seems like we should still have some prose that says something along the lines of "additional attributes (extensions) are added as top-level keys  to this document."

Reply:
Hoping that'll be done by @clemensv's PR